### PR TITLE
bump the terraform version

### PR DESCRIPTION
**Pull request by RagniGarg97**

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/172#" title="Author self-approved">RagniGarg97</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: images/all-in-one/Dockerfile
Comment (mkumatag) [inline images/all-in-one/Dockerfile:7]: Better update with the latest 0.13.x version available

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by mkumatag** [comment]
remove draft when it is ready for review

/cc @Rajalakshmi-Girish

**Comment by Rajalakshmi-Girish** [comment]
> 
> 
> _No description provided._

@mkumatag we plan to have Terraform plugins installation as part of the `all-in-one` image as this is repeatedly needed by all the conformance jobs that use Terraform. Hope it is fine.
I am verifying the changes made by Ragni. Will remove draft once done.

**Comment by mkumatag** [comment]
> > 
> > 
> > _No description provided._
> 
> @mkumatag we plan to have Terraform plugins installation as part of the `all-in-one` image as this is repeatedly needed by all the conformance jobs that use Terraform. Hope it is fine.
> I am verifying the changes made by Ragni. Will remove draft once done.

Okay

**Pull request closed without merging**
